### PR TITLE
Manual Site Import for Large Sites and Custom Upstreams

### LINKED ISSUE #957: Importing a Large Site Doc Update
Re: [Importing a Large Site](https://pantheon.io/docs/articles/sites/migrate/importing-a-large-site)
## Issue Description:
Doc should be revised so it is not specific to large sites. This migration method is for any import outside of the Dashboard import tools available during site creation. 
## Suggested Resolution 
Change title. Update URLs throughout. Reframe the doc so it's not for just large sites, but also for sites that would like to preserve existing history, use a custom distro, etc.

### PR DESCRIPTION
**Pull request by ari-gold**
Closes #957

### CODE REVIEW

**Comment by rachelwhitton** [comment]
Great update @ari-gold, thanks!

This update requires a redirect from [Importing a Large Site](https://pantheon.io/docs/articles/sites/migrate/importing-a-large-site/) --> [Migrate to Pantheon: Manual Site Import](https://pantheon.io/docs/articles/sites/migrate/manual-site-import) once deployed. Local links updated, linkchecker returned with no associated errors. 


+1

File: source/docs/articles/sites/migrate/manual-site-import.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+**Codebase** - all executable code, including core, custom and contrib modules or plugins, themes, and libraries.
+
+As long as you've chosen the same codebase (Drupal 7, Commerce Kickstart, etc.) as the starting point of your Pantheon site, you can use Git to import your existing code and commit history. If you don’t have a Git version controlled codebase, the following will still work.
+
+1. Go to your code directory within your terminal.
+2. If your existing site code is not version controlled with Git, run: ```git init```
+3. From the Site Dashboard, go to the **Dev** environment.
+4. Switch the site's connection mode from SFTP to Git.
+5. Get the upstream's Git connection string:
```
Comment (bmackinney) [inline source/docs/articles/sites/migrate/manual-site-import.md]: If they have already created the site, would it not be simpler to use the site repo in steps 5-6? Something along the lines of:
1. git clone ssh:codeserver-uuid.git site
2. merge code locally (add custom and contrib to repo) and commit.
3. Push

Thoughts?

File: source/docs/articles/sites/migrate/manual-site-import.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+read -sp "Your Pantheon Password: " PASSWORD
+if [[ -z "$PASSWORD" ]]; then
+echo "Woops, need password"
+exit
+fi
+
+while [ 1 ]
+do
+sshpass -p "$PASSWORD" rsync --partial -rlvz --size-only --ipv4 --progress -e 'ssh -p 2222' $ENV.$SITE@appserver.$ENV.$SITE.drush.in:files/* ./files/
```
Comment (bmackinney) [inline source/docs/articles/sites/migrate/manual-site-import.md]: This is missing the required temp-dir flag. see rsync doc.

File: source/docs/articles/sites/migrate/manual-site-import.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+
+You'll need a .sql file containing the data from the site you want to import. If you haven't done so already, make sure you remove any data from the cache tables. That will make your .sql file much smaller and your import that much quicker.
+
+
+1. From the Dev environment on the site Dashboard, click **Connection Info** and copy the Database connection string. It will look similar to this:
+
+ ```
+ mysql -u pantheon -p{massive-random-pw} -h dbserver.dev.{site-id}.drush.in -P {site-port} pantheon
+ ```
+2. From terminal, `cd` into the directory containing your `.sql` archive. Paste the connection string and ammend it with:
```
Comment (bmackinney) [inline source/docs/articles/sites/migrate/manual-site-import.md]: ammend -> append

File: source/docs/articles/sites/migrate/manual-site-import.md (hunk 1 above)
Comment (rachelwhitton) [inline source/docs/articles/sites/migrate/manual-site-import.md]: @bmackinney step 1 of your comment sounds similar to a subtree merge, which clones a repo into a subdir of another repo. Don't think we want pantheon upstream in a subdir of the source site. 

My only question is.. are there negatives to pulling in newly created site repo vs the actual upstream repo? If it doesn't affect future upstream updates I don't see an issue with suggesting they run `git pull --no-rebase --squash -Xtheirs <site repo git string> master`

**Comment by rachelwhitton** [comment]
@bmackinney can you review 652a5ac and 88d79ac? 

I think we're close to :ship:

File: source/docs/articles/sites/migrate/manual-site-import.md [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+Starting from scratch allows your site to connect to that upstream so you can later [apply upstream updates](/docs/articles/sites/code/applying-upstream-updates/) from your Dashboard with one click.
+
+## Import the Codebase
+
+**Codebase** - all executable code, including core, custom and contrib modules or plugins, themes, and libraries.
+
+As long as you've chosen the same codebase (Drupal 7, Commerce Kickstart, etc.) as the starting point of your Pantheon site, you can use Git to import your existing code and commit history. If you don’t have a Git version controlled codebase, the following will still work.
+
+1. Navigate to your existing site's code directory in a local terminal. If your existing code is not version controlled with Git, run:
+
```
Comment (bmackinney) [inline source/docs/articles/sites/migrate/manual-site-import.md]: Indicate requirement: git is installed

File: source/docs/articles/sites/migrate/manual-site-import.md [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+ ```bash
+ git init
+ ```
+2. From the Dev environment of the site Dashboard, set the site's [connection mode](/docs/articles/getting-started/#interact-with-your-code) to Git.
+3. Copy the SSH URL for the site repository, found in the [clone command](/docs/articles/local/starting-with-git/#step-2-copy-the-git-clone-command). **Do not copy `git clone` or the site name.** The URL should look similar to the following:
```
Comment (bmackinney) [inline source/docs/articles/sites/migrate/manual-site-import.md]: `git clone` is not in the string. The site name is. Easier to delete after pasting.

File: source/docs/articles/sites/migrate/manual-site-import.md (hunk 4 above)
Comment (rachelwhitton) [inline source/docs/articles/sites/migrate/manual-site-import.md]: Git is noted as a requirement above on line 15

**Pull request merged**